Tests/AppTests/Utilities.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+@testable import App
+@testable import Vapor
+
+extension Droplet {
+    static func testable() throws -> Droplet {
+        let drop = try Droplet(arguments: ["vapor", "prepare"])
+        try App.setup(drop)
+        try drop.runCommands()
+        return drop
+    }
+
+    // Must be served to activate
+    static func live() throws -> Droplet {
+        let drop = try Droplet(arguments: ["vapor", "serve"])
+        try App.setup(drop)
+        return drop
+    }
+
+    func serveInBackground(_ server: ServerConfig?) throws {
+        background {
```

Comment:
I've found that the `DispatchGroup` wait + 100-200 milliseconds console wait works more quickly and reliably.